cmd/build/helmify/static/README.md

```diff
@@ -8,6 +8,8 @@
 | constraintViolationsLimit | The maximum # of audit violations reported on a constraint  | `20`                                                                      |
 | auditFromCache            | Take the roster of resources to audit from the OPA cache    | `false`                                                                   |
 | disableValidatingWebhook  | Disable ValidatingWebhook                                   | `false`                                                                   |
+| emitDenyEvents            | Emit K8s events in gatekeeper namespace for admission denies| `false`                                                                   |
```

Comment:
nit: Can this be renamed to emitAdmissionEvents as dryrun violations are also recorded for admission requests. Deny sounds only deny events are recorded.

Reply:
+1

Reply:
done